### ISSUE TITLE
【社区治理】SigureMo 发起 Committer 身份申请

### ISSUE THREAD

**Issue by SigureMo** [PaddlePaddle/Paddle#45841]
## 基本信息

| 申请人 GitHub ID | **Paddle Repo 整体 merge PR 数**                                                           | **Paddle Repo 整体 review PR 数**                                                                                  | **Paddle Repo 整体报告 Issue 数**                                                   |
| ---------------- | ------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------- |
| SigureMo         | [22](https://github.com/PaddlePaddle/Paddle/pulls?q=is%3Apr+is%3Amerged+author%3ASigureMo) | [11](https://github.com/PaddlePaddle/Paddle/pulls?q=is%3Apr+is%3Amerged+reviewed-by%3ASigureMo+-author%3ASigureMo) | [3](https://github.com/PaddlePaddle/Paddle/issues?q=is%3Aissue+author%3ASigureMo) |

## 社区贡献概况

以下是此前在本 repo 里做过的贡献：

- 视觉模型 backbone API
- 修复部分 API 文档及一些 typo 之类的
- 单测报错信息优化
- review 部分 API 文档（含部分文档修复和部分 Hackathon 任务）

至于理由嘛，主要是 review 时有时候让贡献者自己改太麻烦了，有时候往往需要沟通很多遍才能达到满意的效果（比如修改部分有遗漏和代码风格等问题）

### “高质量” merged PR 展示

| PR 号                                  | PR 标题                                                                                         | PR 简介                                     | Reviewer                               |
| -------------------------------------- | ----------------------------------------------------------------------------------------------- | ------------------------------------------- | -------------------------------------- |
| #36034, #36064, #36067, #36070, #36952 | [PaddlePaddle Hackathon] add GoogLeNet, ...                                                     | 第一期 Hackathon 任务                       | LielinJiang, XiaoguangHu01, TCChenlong |
| #38653                                 | add MobileNetV3                                                                                 | MobileNetV3                                 | LielinJiang, XiaoguangHu01, TCChenlong |
| #43537                                 | fix paddle.Model en docs                                                                        | 修复 paddle.Model 文档                      | Ligoml                                 |
| #43636                                 | fix paddle.vision.models.\* en docs                                                             | 修复 paddle.vision.models 下全部 API 文档   | Ligoml                                 |
| #43649                                 | fix paddle.vision.datasets.\* en docs                                                           | 修复 paddle.vision.datasets 下全部 API 文档 | Ligoml                                 |
| #44947, #44988, #45213, #45251         | \[CodeStyle\] use np.testing.assert_array_equal instead of self.assertTrue(np.array_equal(...)) | 单测报错信息优化                        | luotao1, XiaoguangHu01, qili93         |

### “高质量” review PR 展示

| PR 号  | PR 标题                                                                             | PR 简介               | review 详情                      |
| ------ | ----------------------------------------------------------------------------------- | --------------------- | -------------------------------- |
| #40719 | fix paddle.vision.transforms.Resize en docs                                         | 文档修复              | 文档示例代码建议等               |
| #43996 | 【Fix en docs】 Fix document formatting                                             | 文档修复              | 文档格式建议、CI 建议等          |
| #44568 | 【PaddlePaddle Hackathon 3 No.17】为 Paddle 新增 sgn 文档                           | 第三期 Hackathon 任务 | 文档中英文一致性建议、格式建议等 |
| #44741 | 【PaddlePaddle Hackathon 3 No.16】为 Paddle 新增 API paddle.take                    | 第三期 Hackathon 任务 | 文档格式建议等                   |
| #45068 | update loss.py in nn.functional and nn.layer module                                 | 文档修复              | 文档格式建议等                   |
| #45168 | \[triu_indices\] add triu_indices_op                                                | 第三期 Hackathon 任务 | CI 建议、文档参数建议等          |
| #45266 | 【PaddlePaddle Hackathon 3 No.14】为 Paddle 新增 remainder\_ API                    | 第三期 Hackathon 任务 | 单测建议等                       |
| #45423 | update PixelShuffle and PRelu api in vision.py and activation.py；test=document_fix | 文档修复              | 文档示例代码建议等               |

### 代表性 Issue 展示

| Issue 号 | Issue 标题                                                             | Issue 简介                                                                              | 解决情况                                                |
| -------- | ---------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------- |
| #35891   | 使用 shape 为 (1, ) 的 Tensor 进行索引时会使结果少一根轴               | 如题，根本原因应该是 Paddle 目前会将仅含一个元素的 1-D Tensor 视作 Scalar（0-D Tensor） | 需支持 0-D Tensor，may be fixed by #44995               |
| #43766   | macOS (M1) 编译 develop 分支失败                                       | 如题                                                                                    | fixed by #42325                                        |
| #44671   | \[Docs\] 中文文档部分 API 文件在大小写不敏感的文件系统下的相互覆盖问题 | 在大小写不敏感的文件系统下，由于同名文件导致的问题                                      | fixed by https://github.com/PaddlePaddle/docs/pull/5174 |

#### 担保人意见

@Ligoml, @luotao1

**Comment by luotao1** [comment]
+1

@SigureMo 在 [单测优化报错项目](https://github.com/PaddlePaddle/community/blob/master/pfcc/call-for-contributions/code_style_improvement_for_unittest.md) 中作为组长，带领小组存量（批量自动化）共修复了400+文件近2700+处单测的断言函数，增量使用CI检查项阻止新增不合规断言函数的出现，大幅提升了飞桨单测的报错信息丰富度。

**Comment by XiaoguangHu01** [comment]
+1

**Comment by Ligoml** [comment]
同意~

**Comment by raindrops2sea** [comment]
代表飞桨再次感谢SigureMo的突出贡献！已添加对应权限，请注意查看:-)

**Comment by SigureMo** [comment]
> 代表飞桨再次感谢SigureMo的突出贡献！已添加对应权限，请注意查看:-)

已收到邀请～感谢～(๑>؂<๑）
